### ISSUE TITLE
Please put correct permissions and reload install process

### ISSUE THREAD

**Issue by lsv** [thelia/thelia-project#33]
Just for us that doesnt know everything.

I would recommend to change "reload install process" to ```composer run-script post-create-project-cmd```, so we dont need to open the composer.json file to see which commands there is.

Because, "reload install process" this sounds like "run the same command again", which means I would run ```composer create-project thelia/thelia-project thelia``` which ofcourse fails, because the directory is not empty anymore.

**Comment by bretto36** [comment]
yes this confused me, i had to delete the folder and rerun because i didn't know which command i needed to run after fixing the issues
